src/main/java/com/mongodb/hibernate/jdbc/MongoConnection.java

```diff
@@ -46,7 +46,7 @@ final class MongoConnection implements ConnectionAdapter {
 
     private final Logger logger = LoggerFactory.getLogger(MongoConnection.class);
 
-    // temporary hard-coded database prior to the db config tech design finalizing
+    // TODO-HIBERNATE-38 temporary hard-coded database prior to the db config tech design finalizing
```

Comment:
The benefit of the approach is most obvious with this comment. Previously, the comment was undiscoverable, and there was no way to unequivocally refer to it from the corresponding [HIBERNATE-38](https://jira.mongodb.org/browse/HIBERNATE-38) ticket.

Reply:
I adopt the approach without any reserve